### PR TITLE
Added functionality for two live data plots

### PR DESCRIPTION
**Pull request by roninsharma25**

### CODE REVIEW

File: GUI/widgets/plot_buttons.py [hunk 1]
```diff
@@ -23,8 +26,18 @@ def initUI(self):
         self.example_button.setText("Example Button")
         self.example_button.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
 
+        self.plot_dropdown = QComboBox()
+        self.plot_dropdown.setEditable(True)
+        self.plot_dropdown.lineEdit().setAlignment(Qt.AlignCenter)
+        self.plot_dropdown.lineEdit().setReadOnly(True)
+        for plot_name in self.plot_names:
+            self.plot_dropdown.addItem(plot_name)
+        self.plot_dropdown.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1;")
```
Comment (ryanwmao) [inline GUI/widgets/plot_buttons.py:35]: is it possible to set css of the dropdown from a separate file?

**Review by ryanwmao — COMMENTED** [review]
what other data will we be plotting?

**Comment by roninsharma25** [comment]
> what other data will we be plotting?

@ryanwmao Each sensor will send data to the GUI. We've initially focused on plotting data from thermistors but there are a bunch of sensors computing systems are currently working with. Each sensor will have its own plot.

File: GUI/widgets/plot_buttons.py (hunk 1 above)
Comment (roninsharma25) [reply GUI/widgets/plot_buttons.py:35]: Most likely, I was planning on batching all those changes as part of ticket 237

**Review by ryanwmao — APPROVED** [review]

**Pull request merged**